.travis.yml

```diff
@@ -7,13 +7,13 @@
 
 language: node_js
 node_js:
-  - "10"
+  - '10'
 
 jobs:
   include:
     - stage: test
       script:
-        - echo "This is just a test"
+        - yarn test
```

Comment:
Looks like we are using npm (package.lock exists, but yarn.lock doesn't). Maybe we should use `npm test` here